ui/js/controllers/perf/graphs.js

```diff
@@ -255,7 +261,9 @@ perf.controller('GraphsCtrl', [
             $("#overview-plot").bind("plotselected", function (event, ranges) {
                 deselectDataPoint();
                 hideTooltip();
-
+                if ($scope.selectedDataPoint) {
+                    showTooltip($scope.selectedDataPoint);
+                }
```

Comment:
You can delete these lines, they don't do anything (since you just scheduled deselection above)